requirements.yml

```diff
@@ -16,7 +16,7 @@
 
 - name: openmicroscopy.omero-server
   src:  https://github.com/openmicroscopy/ansible-role-omero-server.git
```

Comment:
Should these all come from Galaxy, rather than GH now?

Reply:
Yes

Reply:
Are you pushing this, @kennethgillen ?

Reply:
Yes, will pull the GH urls. (Does it apply to all requirements?)

Reply:
Ah, @manics already replied to my first question which was "all". Pushing.